Resources/Prototypes/Floof/Loadouts/head.yml

```
@@ -20,3 +20,130 @@
          - Security
          - Command
 
+- type: loadout
+  id: LoadoutHeadHatAnimalCat
+  category: Head
+  cost: 2
+  exclusive: true
+  items:
+    - ClothingHeadHatAnimalCat
+
+- type: loadout
+  id: LoadoutHeadHatAnimalCatBrown
+  category: Head
+  cost: 2
+  exclusive: true
+  items:
+    - ClothingHeadHatAnimalCatBrown
+
+- type: loadout
+  id: LoadoutHead
```

Prettysure its missing something.
```suggestion
  id: LoadoutHeadAnimalCatBlack
```

---

As you can tell I had a slow descent into insanity today :3